extensions/wikia/ContentReview/ContentReviewHelper.php

```diff
@@ -239,113 +237,6 @@ public function hasPageApprovedId( CurrentRevisionModel $model, $wikiId, $pageId
 		return ( !empty( $currentData ) && (int)$currentData['revision_id'] === $oldid );
 	}
 
-	public function shouldDisplayReviewerToolbar() {
-		global $wgCityId;
-
-		$title = $this->getTitle();
-		$contentReviewRequest = $this->getRequest()->getBool( self::CONTENT_REVIEW_PARAM );
-
-		if ( $title->inNamespace( NS_MEDIAWIKI )
-			&& $contentReviewRequest
-			&& $title->isJsPage()
-			&& $title->userCan( 'content-review' )
-		) {
-			$reviewModel = new ReviewModel();
-
-			$diffRevisionId = $this->getRequest()->getInt( 'diff' );
-			$articleId = $title->getArticleID();
-			$diffRevisionInfo = $reviewModel->getRevisionInfo(
-				$wgCityId,
-				$articleId,
-				$diffRevisionId
-			);
-
-			$status = (int)$diffRevisionInfo['status'];
-
-			// Always make sure it's in review if this is a content review request
-			if ( $status === ReviewModel::CONTENT_REVIEW_STATUS_UNREVIEWED ) {
-				$reviewerId = $this->getUser()->getId();
-				try {
-					$reviewModel->updateRevisionStatus( $wgCityId, $articleId, $status,
-						ReviewModel::CONTENT_REVIEW_STATUS_IN_REVIEW, $reviewerId );
-				} catch ( \FluentSql\Exception\SqlException $e ) {
-					// Master-slave replication has not finished, ignore
-				}
-
-				return true;
-			}
-
-			return ( $status === ReviewModel::CONTENT_REVIEW_STATUS_IN_REVIEW );
-		}
-
-		return false;
-	}
-
-	/**
-	 * Returns an HTML with a toolbar displayed to reviewers.
-	 * @param int $revisionId An ID of the revision that is currently being reviewed
-	 * @return string
-	 * @throws \Exception
-	 */
-	public function getToolbarTemplate( $revisionId ) {
-		global $wgCityId;
-
-		return \MustacheService::getInstance()->render(
-			self::CONTENT_REVIEW_TOOLBAR_TEMPLATE_PATH,
-			[
-				'toolbarTitle' => wfMessage( 'content-review-diff-toolbar-title' )->plain(),
-				'wikiId' => $wgCityId,
-				'pageId' => $this->getTitle()->getArticleID(),
-				'approveStatus' => ReviewModel::CONTENT_REVIEW_STATUS_APPROVED,
-				'buttonApproveText' => wfMessage( 'content-review-diff-approve' )->plain(),
-				'rejectStatus' => ReviewModel::CONTENT_REVIEW_STATUS_REJECTED,
-				'buttonRejectText' => wfMessage( 'content-review-diff-reject' )->plain(),
-				'talkpageUrl' => $this->prepareProvideFeedbackLink( $this->getTitle(), $revisionId ),
-				'talkpageLinkText' => wfMessage( 'content-review-diff-toolbar-talkpage' )->plain(),
-				'guidelinesUrl' => wfMessage( 'content-review-diff-toolbar-guidelines-url' )->useDatabase( false )->plain(),
-				'guidelinesLinkText' => wfMessage( 'content-review-diff-toolbar-guidelines' )->plain(),
-			]
-		);
-	}
-
-	/**
-	 * Link for adding new section on script talk page. Prefilled with standard explanation of rejection.
-	 * @param \Title $title Title object of JS page
-	 * @param int $revisionId
-	 * @return string full link to edit page
-	 */
-	public function prepareProvideFeedbackLink( \Title $title, $revisionId = 0 ) {
```

Comment:
This is still called in ContentReviewApiController on the helper object, so that would cause a fatal.